### PR TITLE
Fixes #21037: Remove create-node-api menu entry and page

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/21037
image

### CODE REVIEW

File: create-node-api/src/main/scala/com/normation/plugins/createnodeapi/CreateNodeApiPluginDef.scala [hunk 1]
```diff
@@ -64,13 +64,13 @@ class CreateNodeApiPluginDef(override val status: PluginStatus) extends DefaultP
...
 
-  override def pluginMenuEntry: Option[Menu] = {
-    Some(Menu("createNodeApi", <span>Create node API</span>) /
-      "secure" / "plugins" / "CreateNodeAPI"
-      >> LocGroup("pluginsGroup")
-      >> TestAccess ( () => Boot.userIsAllowed("/secure/index", Administration.Read))
-      >> Template(() => ClasspathTemplates("template" :: "CreateNodeApiManagement" :: Nil ) openOr <div>Template not found</div>)
-    )
-  }
+//  override def pluginMenuEntry: Option[Menu] = {
```
Comment (amousset) [inline create-node-api/src/main/scala/com/normation/plugins/createnodeapi/CreateNodeApiPluginDef.scala:67]: we can completely remove it imo

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by VinceMacBuche** [comment]
OK, squash merging this PR

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**